web/index.html

```diff
@@ -622,21 +754,21 @@ <h3>Used in</h3>
                     this.showToast = false;
                 }
                 // Load recipes from JSON file
-                fetch(DATA_PREFIX + 'index.json')
+                fetch(DATA_PATH + 'index.json')
                     .then(response => { console.log(response); return response.json() })
                     .then(index => {
                         this.data.index = Object.freeze(Object.fromEntries(Object.entries(index).map(x => [x[0], [x[1][0], x[1][1]]])))
                         this.data.craftingCache = Object.freeze(Object.fromEntries(Object.entries(index).map(x => [x[0], [x[1][3], x[1][4]]])))
                         this.data.list = Object.freeze(Object.entries(index).sort((a, b) => a[1][2] - b[1][2]).filter(x => x[1][1] != "Nothing").map(x => [x[0], x[1][1]]))
                         this.data.costs = Object.freeze(Object.fromEntries(Object.entries(index).map(x => [x[0], x[1][2]])))
 
-                        this.navigateTo(getQuery().e || "");
+                        this.navigateTo(getQuery().e || (document.referrer === HELPER_PATH ? localStorage.getItem("lastElement") : "") || "");
```

Comment:
If this page is being navigated to from the Helper page, that means that the user just ran the bookmarklet to get their data. When they navigate back to the site like this then we want them to go back to whatever their previous element was. Because of some annoyances of bookmarklets that need to run multiple times independently and lose state, the easiest way to do this is by just saving the last element in local storage and then loading from there only when the user is navigating back using the bookmarklet